### PR TITLE
ISPN-11503 Export Rolling Upgrade operations via REST

### PR DESCRIPTION
**Pull request by gustavonalle**
https://issues.redhat.com/browse/ISPN-11503

### CODE REVIEW

File: persistence/remote/src/main/java/org/infinispan/persistence/remote/configuration/RemoteStoreConfiguration.java [hunk 1]
```diff
@@ -58,16 +65,16 @@ public static AttributeSet attributeDefinitionSet() {
...
    private final Attribute<Boolean> tcpNoDelay;
    private final ConnectionPoolConfiguration connectionPool;
    private final ExecutorFactoryConfiguration asyncExecutorFactory;
    private final SecurityConfiguration security;
+   private List<RemoteServerConfiguration> servers;
 
    public RemoteStoreConfiguration(AttributeSet attributes, AsyncStoreConfiguration async,
                                    ExecutorFactoryConfiguration asyncExecutorFactory, ConnectionPoolConfiguration connectionPool,
-                                   SecurityConfiguration security) {
+                                   SecurityConfiguration security, List<RemoteServerConfiguration> servers) {
```
Comment (karesti) [inline persistence/remote/src/main/java/org/infinispan/persistence/remote/configuration/RemoteStoreConfiguration.java]: So we will have access to other Infinispan Clusters from the console ?

**Comment by karesti** [comment]
Looks good but I need to see how can we add this feature to the console

File: persistence/remote/src/main/java/org/infinispan/persistence/remote/configuration/RemoteStoreConfiguration.java (hunk 1 above)
Comment (gustavonalle) [reply persistence/remote/src/main/java/org/infinispan/persistence/remote/configuration/RemoteStoreConfiguration.java]: If you get the server config from REST, those servers will be listed, but they are the servers configured in the remote store for a cache

Comment (gustavonalle) [reply persistence/remote/src/main/java/org/infinispan/persistence/remote/configuration/RemoteStoreConfiguration.java]: and those server are going to be shutdown after the rolling upgrade, not sure how useful it is

**Pull request merged**